react/src/attention/AttentionPattern.tsx

```diff
@@ -164,15 +165,37 @@ export function AttentionPattern({
 
   return (
     <Col>
-      <Row style={{ aspectRatio: showAxisLabels ? undefined : "1/1" }}>
```

Comment:
Why was this previously undefined for showAxisLabels==true?

Reply:
I'm not sure. This is definitely a bit of a Chesterton's fence. I have tried with `showAxisLabels` set to `true` and `false` so I think it should be okay.